app/src/main/res/layout/topic_summary_view.xml

```diff
@@ -10,59 +11,88 @@
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="140dp"
+    android:layout_width="match_parent"
     android:layout_height="160dp"
-    android:layout_margin="10dp"
+    android:layout_marginStart="8dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginBottom="20dp"
+    android:elevation="4dp"
     app:roundedRectDrawableWithColor="@{viewModel.backgroundColor}"
     android:clickable="true"
     android:focusable="true"
     android:onClick="@{viewModel::clickOnSummaryTile}">
 
     <ImageView
-      android:layout_width="0dp"
-      android:layout_height="0dp"
-      android:src="@{viewModel.topicSummary.topicThumbnail.thumbnailGraphic}"
-      android:scaleType="centerCrop"
-      android:paddingStart="20dp"
-      android:paddingEnd="20dp"
-      android:paddingTop="20dp"
+      android:id="@+id/imageView"
+      android:layout_width="match_parent"
+      android:layout_height="90dp"
```

Comment:
This is not in multiple of 4, suggestion making this 92.

Reply:
Done.